### DISCUSSION TITLE
希望能够自动删除旧版mobile_v**.sh文件

### DISCUSSION THREAD

**Discussion by Qs315490** [lollipopkit/flutter_server_box#339] [Ideas]
使用这个软件会在当前用户文件夹下生成 `.config/server_box/mobile_v版本号.sh` 文件，经历几次软件更新后，这个目录下的文件会越积越多，最好能实现保留最新版本删除旧版文件

**Comment by lollipopkit** [comment]
感谢反馈，我也想到过这个问题。

但是可能存在不同版本的app同时连接服务器，删除脚本会导致其无法正常运行。就算同时存在100个脚本，也只占400k。所以暂时不考虑该问题，后继可能会解决。
